api/src/score/ro/raw-data.ro.ts

```diff
@@ -39,16 +40,20 @@ export class RawDataRo {
   }
   convertApplicationToContent(data: Application[]) {
     const content = data.map((item: Application) => {
+      const formattedDate = dayjs(item.updatedAt).isValid()
+        ? dayjs(item.updatedAt).format('YYYY-MM-DD')
```

Comment:
Sorry, I forgot to mention this in previous PR as well, in AM/PM format I think. It's good to confirm though.

Reply:
Is there a way to process date on FE? So we don't have to add a new package for api

Reply:
> Is there a way to process date on FE? So we don't have to add a new package for api

Unfortunately it would require us to loop through the object again on the FE side, I put it here since we're already doing a map and figured dayjs was compact enough to just add it to the api side.  I tried adding it to root, but I think I need to modify the way docker is copying the package

Reply:
Alright, but we still need to update date/time format

Reply:
we can confirm in standup, the tickets just say YYYY-MM-DD, so I was just going based off that